### DISCUSSION TITLE
Error while running nx

### DISCUSSION THREAD

**Discussion by Ahmetadm** [nrwl/nx#9325] [General]
Hi everyone, 
Can someone help me about below error i get while running 'nx serve' command ?
i did not ever run  "sudo" or "root" as part of your install/setup.

Thank you in advance

yarn run v1.22.17
$ yarn nx serve
$ cross-env NODE_OPTIONS=--max-old-space-size=3333 nx serve

> nx run web:serve

info  - Loaded env from apps/web/.env.development
warn  - You have enabled experimental feature(s).
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use them at your own risk.

EACCES: permission denied, mkdir '../../dist'

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  NX   Running target "web:serve" failed

   Failed tasks:
   
   - web:serve
   
   Hint: run the command with --verbose for more details.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
username@username.com:~/project$

**Comment by EhsanHerai** [comment]
`EACCES: permission denied, mkdir '../../dist'`

as it seems you have a permission issue.
try to use `chmod` to grant directory creation access

**Comment by rfprod** [comment]
@Ahmetadm So, you say you execute the `serve` command from this directory
```bash
user@computer:~/project$
```
that is probably equivalent to smth like
```bash
user@computer: /home/user/project$
```

and the CLI tries to perform an operation on `../../dist`?

I wonder if the CLI tries to create a `dist` folder in the `home` directory.

Logging `cwd` before the `mkdir` command might have helped.
